src/lib/fs/files.h

```diff
@@ -103,4 +103,38 @@ char *read_file_to_str_until_eof(int fd, size_t max_bytes_to_read,
                                  size_t *sz_out)
   ATTR_MALLOC;
 
+#if !defined(HAVE_GETDELIM) || defined(TOR_UNIT_TESTS)
+ssize_t compat_getdelim_(char **lineptr, size_t *n, int delim, FILE *stream);
```

Comment:
There's no function comment for getdelim() - do we expect developers to look it up in the relevant man page?

Reply:
I think that's best -- since this is a POSIX standard, if our function differs from the manpage, one of them is wrong.